src/libraries/System.Text.Json/src/System/Text/Json/ThrowHelper.Serialization.cs

```diff
@@ -459,7 +459,7 @@ public static void ThrowInvalidOperationException_SerializationDuplicateTypeAttr
         [DoesNotReturn]
         public static void ThrowInvalidOperationException_ExtensionDataConflictsWithUnmappedMemberHandling(Type classType, JsonPropertyInfo jsonPropertyInfo)
         {
-            throw new InvalidOperationException(SR.Format(SR.ExtensionDataConflictsWithUnmappedMemberHandling, classType, jsonPropertyInfo.MemberName));
+            throw new InvalidOperationException(SR.Format(SR.ExtensionDataConflictsWithUnmappedMemberHandling, jsonPropertyInfo.MemberName, classType));
```

Comment:
Per:

```
  <data name="ExtensionDataConflictsWithUnmappedMemberHandling" xml:space="preserve">
    <value>The type '{0}' is marked 'JsonUnmappedMemberHandling.Disallow' which conflicts with extension data property '{1}'.</value>
  </data>
```

should you be changing:
https://github.com/dotnet/runtime/blob/b493bd939bbf2440493f4d69ba3b20a166c1ea99/src/libraries/System.Text.Json/src/System/Text/Json/ThrowHelper.Serialization.cs#L593

instead? (UnmappedJsonProperty)

the previous looks correct

Reply:
Yeah, it is, my bad.
It had been fixed in https://github.com/dotnet/runtime/pull/81718/files